widget/slider.go

```diff
@@ -0,0 +1,273 @@
+package widget
+
+import (
+	"image/color"
+	"math"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/canvas"
+	"fyne.io/fyne/theme"
+)
+
+const (
+	maxSliderDecimals = uint8(12)
+	standardScale     = 6
+	minLongSide       = 150
```

Comment:
This seems like a high number for a minimum size? I imagine a minimum size might be closer to the "on/off toggle" seen on some mobile devices as you can see 2 distinct states which this can be used for...

Reply:
yeah it should probably be around 50